setup.py

```diff
@@ -80,14 +81,14 @@ def install_binary_dep(self, url, name, rev_name, binary_name, make_target=""):
         )
 
     def run(self):
-        if sys.base_prefix == sys.prefix:
+        if sys.base_prefix == sys.prefix and not os.getenv("CONDA_PREFIX"):
```

Comment:
Is this enough to close #17 ?

Reply:
I think it might be, but this is a bit experimental and I'm not 100% sure it covers all edge cases.